styles.css

```diff
@@ -692,8 +653,44 @@ textarea {
     background-color: #e5e5e5;
   }
 
-  .close-icon {
+  #projects-section-mobile {
+    display: none;
+  }
+
```

Comment:
Because you are not styling the tags, I don't have a line to reference.

- [ ]  Please style the mobile versions of the tags as in the Figma design:

**Your mobile version:**
![image](https://user-images.githubusercontent.com/83596135/180836646-a865eeb7-93b3-459c-a5c3-c2d19cfd2d9b.png)

**Figma design - mobile version:**
![image](https://user-images.githubusercontent.com/83596135/180837114-2f49f757-6f06-4c7a-9cf9-2752320de210.png)